src/test/setup.ts

```diff
@@ -21,6 +21,8 @@ expect.extend({
   toMatchImageSnapshot,
 });
 
+jest.mock('ally.js/style/focus-within');
```

Comment:
When is ally.js loaded in the real code? I probably missed it while reviewing.

Reply:
Thanks! And it seems to be to support [:focus-within in IE](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within)